modules/bound-sa-tokens-configuring.adoc

```diff
@@ -67,13 +67,32 @@ If the output shows a message similar to one of the following messages, the upda
 ** `3 nodes are at revision 11; 0 nodes have achieved new revision 12`
 ** `2 nodes are at revision 11; 1 nodes are at revision 12`
 
-... Manually restart all pods in the cluster:
+.. Optional: Force the holder to request a new bound token either by performing a rolling node restart or by manually restarting all pods in the cluster.
++
+// TODO: When would a user NOT need to do this?
+
+*** Perform a rolling node restart:
++
+[WARNING]
+====
+If you have workloads running on your cluster, performing a rolling node restart can cause a service interruption. TODO: What to say about how to "be careful" to minimize disruption?
+
+// TODO: Are the warnings different enough between node restarts vs pod deletions to keep them separate, or should we combine them?
```

Comment:
@stlaz @mfojtik This was the question I asked in slack, but I was wondering if the warning is the same no matter which method you choose, so we could combine the warnings. Or if the consequences are different for each, so we should continue to keep them separated

Reply:
They are different. By node restarts you can cause a severe disruption to services that might only be able/allowed to run on specific nodes, whereas generic pod restart will likely only cause a short blip.